Sources/Datadog/Datadog.swift

```diff
@@ -102,6 +102,11 @@ public class Datadog {
         }
     }
 
+    /// Returns `true` if the Datadog SDK is already initialized, `false` otherwise.
+    public static var isInitialized: Bool {
+        instance != nil
```

Comment:
iirc some previous Xcode/Swift versions don't compile when `return` is omitted.
better safe than sorry: can you please add `return` to this line?